maintenance_project/views/maintenance_equipment_views.xml

```diff
@@ -40,10 +40,13 @@
                 <field
                     name="create_project_from_equipment"
```

Comment:
Not sure to understand the UX of this check... So you mark this check, and when you write, the project is created? Why not having a button for that?

Reply:
I totally agree, I do not like this field and it serves only for that purpose (to create a project when the maintenance.equipment is created if this check is checked).

I will make the corresponding change to simplify this behavior.

Reply:
Removed field and added a button.

Reply:
Review refactoring, with this new approach "create project from equipment" flow has been changed:
- Before, the form check enabled creating and initially assigning a new project based on new equipment, only when an equipment is created. Later, the only option was replacing project for other (or leave it empty), never making project clones.
- After, new project based on equipment is available once equipment is created (two steps), and only when project is empty. Indeed, every time equipment project is empty you can create a project, so you could theoretically create more than one with exactly the same name (and that could be confusing).

Not sure how to make new behavior closer to previous one, and still replacing "instrumental" field by an action button (and that's a good idea, agree with it).

Reply:
I don't understand you, David

Reply:
I honestly don't understand the issue with the current change, but perhaps it would be solved changing `readonly` attrs in the `project_id` field if it is already set and the team is created `([[('project_id', '!=', False),('id', '!=', False)])`. 

With this approach:
- Creating the equipment: You will be able to set a project from the existing ones.
- Equipment is already created: You can use the “_Create project_” button (if not already set) or set a project. It will not be possible to “leave empty” the `project_id` field by UI.

What do you think?